client/navigation/connected-left-nav.tsx

```diff
@@ -344,14 +354,15 @@ function ConnectedChatNavEntry({
   channelId: SbChannelId
   onLeave: (channelId: SbChannelId) => void
 }) {
+  const { t } = useTranslation()
   const channelInfo = useAppSelector(s => s.chat.idToBasicInfo.get(channelId))
   const hasUnread = useAppSelector(s => s.chat.unreadChannels.has(channelId))
   const [pathname] = useLocation()
 
   return (
     <ChatNavEntry
       channelId={channelId}
-      channelName={channelInfo?.name ?? 'Loading...'}
+      channelName={channelInfo?.name ?? t('navigation.leftNav.loadingChannel', 'Loading...')}
```

Comment:
This should have an ellipsis instead of "..." (should really be a lint warning :) )